src/helpers/ConfigHelper.ts

```diff
@@ -65,7 +65,18 @@ export class ConfigHelper {
 
   public static doesCurrentOriginMatchConfigOrigin(configOrigin: string): boolean {
     try {
-      return location.origin === new URL(configOrigin).origin;
+      const configUrl = new URL(configOrigin);
```

Comment:
Variable names are mostly good here but still hard to read as the number check being done here. There are a few options to clean this up.
1. Do early returns.
2. Make separate private static function in this file.
3. If generic enough some logic might be a good fit as a Until function.